src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -1341,7 +1368,18 @@ FCIMPL1(void, RhpReversePInvoke, ReversePInvokeFrame * pFrame)
     if (pCurThread->InlineTryFastReversePInvoke(pFrame))
         return;
 
-    RhpReversePInvokeAttachOrTrapThread2(pFrame);
+    RhpReversePInvokeAttachOrTrapThread2(pFrame, true);
+}
+FCIMPLEND
+
+FCIMPL1(void, RhpReversePInvokeNoModuleInitializer, ReversePInvokeFrame * pFrame)
+{
+    Thread * pCurThread = ThreadStore::RawGetCurrentThread();
+    pFrame->m_savedThread = pCurThread;
+    if (pCurThread->InlineTryFastReversePInvoke(pFrame))
```

Comment:
The Inline portion will always fail here since the thread is not supposed to be attached, right?

Reply:
This was a copypaste from the other transition, but maybe we don't need this.